possibility to set namespaces manually when the autodiscover featre is disabled by the organization

possibility to add namespaces manually when the autodiscover feature is disabled by the organization:
com.redhat.devtools.intellij.kubernetes.model.util.ResourceException: Could not get Namespaces for server https://myorganization.com

like this? 

```
com.redhat.devtools.intellij.kubernetes.model.util.ResourceException: Could not get Namespaces for server https://api.xxxxx.k8s.eu-central-1.aws.xxxxxxxx.com/
	at com.redhat.devtools.intellij.kubernetes.model.context.ActiveContext.getAllResources(ActiveContext.kt:148)
	at com.redhat.devtools.intellij.kubernetes.model.context.ActiveContext.getCurrentNamespace(ActiveContext.kt:122)
	at com.redhat.devtools.intellij.kubernetes.model.ResourceModel.getCurrentNamespace(ResourceModel.kt:106)
	at com.redhat.devtools.intellij.kubernetes.tree.KubernetesDescriptors$NamespacesFolderDescriptor.getSubLabel(KubernetesDescriptors.kt:126)
	at com.redhat.devtools.intellij.kubernetes.tree.KubernetesDescriptors$NamespacesFolderDescriptor.getSubLabel(KubernetesDescriptors.kt:114)
	at com.redhat.devtools.intellij.kubernetes.tree.TreeStructure$Descriptor.update(TreeStructure.kt:331)
	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.getUpdatedPresentation(PresentableNodeDescriptor.java:88)
	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.update(PresentableNodeDescriptor.java:32)
	at com.intellij.ui.tree.StructureTreeModel$Node.update(StructureTreeModel.java:486)
	at com.intellij.ui.tree.StructureTreeModel$Node.<init>(StructureTreeModel.java:467)
	at com.intellij.ui.tree.StructureTreeModel$Node.<init>(StructureTreeModel.java:460)
	at com.intellij.ui.tree.StructureTreeModel.getValidChildren(StructureTreeModel.java:418)
	at com.intellij.ui.tree.StructureTreeModel.validateChildren(StructureTreeModel.java:329)
	at com.intellij.ui.tree.StructureTreeModel.getNode(StructureTreeModel.java:323)
	at com.intellij.ui.tree.StructureTreeModel.getChildren(StructureTreeModel.java:343)
	at com.intellij.ui.tree.AsyncTreeModel$CmdGetChildren.computeNode(AsyncTreeModel.java:613)
	at com.intellij.ui.tree.AsyncTreeModel$Command.computeNode(AsyncTreeModel.java:489)
	at com.intellij.util.concurrency.Invoker$Task.run(Invoker.java:381)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:192)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:610)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:685)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:641)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:609)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:78)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:179)
	at com.intellij.util.concurrency.Invoker.startTask(Invoker.java:236)
	at com.intellij.util.concurrency.Invoker.invokeSafely(Invoker.java:194)
	at com.intellij.util.concurrency.Invoker.lambda$offerSafely$0(Invoker.java:177)
	at com.intellij.util.concurrency.Invoker$Background.lambda$offer$0(Invoker.java:508)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:244)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:30)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.executeFirstTaskAndHelpQueue(BoundedTaskExecutor.java:222)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:218)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:210)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://api.xxxxx.k8s.eu-central-1.aws.xxxxxxxx.com/api/v1/namespaces. Message: Forbidden! User stefano-xxxxx-id doesn't have permission. namespaces is forbidden: User "stefano.xxxxx@xxxxxxx.com" cannot list resource "namespaces" in API group "" at the cluster scope.
	at io.fabric8.kubernetes.client.KubernetesClientException.copyAsCause(KubernetesClientException.java:238)
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.waitForResult(OperationSupport.java:538)
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.list(BaseOperation.java:427)
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.list(BaseOperation.java:392)
	at io.fabric8.kubernetes.client.dsl.internal.BaseOperation.list(BaseOperation.java:93)
	at com.redhat.devtools.intellij.kubernetes.model.resource.NonNamespacedResourceOperator.loadAllResources(NonNamespacedResourceOperator.kt:51)
	at com.redhat.devtools.intellij.kubernetes.model.resource.NonNamespacedResourceOperator.getAllResources(NonNamespacedResourceOperator.kt:42)
	at com.redhat.devtools.intellij.kubernetes.model.resource.NonNamespacedResourceOperator.getAllResources(NonNamespacedResourceOperator.kt:34)
	at com.redhat.devtools.intellij.kubernetes.model.context.ActiveContext.getAllResources(ActiveContext.kt:141)
	... 40 more
Caused by: io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://api.xxxxx.k8s.eu-central-1.aws.xxxxxxxx.com/api/v1/namespaces. Message: Forbidden! User stefano.xxxxx-id doesn't have permission. namespaces is forbidden: User "stefano.xxxxx@xxxxxx.com" cannot list resource "namespaces" in API group "" at the cluster scope.
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:728)
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.requestFailure(OperationSupport.java:708)
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.assertResponseCode(OperationSupport.java:657)
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$handleResponse$0(OperationSupport.java:587)
	at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
	at io.fabric8.kubernetes.client.dsl.internal.OperationSupport.lambda$retryWithExponentialBackoff$2(OperationSupport.java:629)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863)
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
	at io.fabric8.kubernetes.client.http.StandardHttpClient.lambda$withUpstreamCancellation$3(StandardHttpClient.java:100)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863)
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
	at io.fabric8.kubernetes.client.http.ByteArrayBodyHandler.onBodyDone(ByteArrayBodyHandler.java:52)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:863)
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:841)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2147)
	at io.fabric8.kubernetes.client.okhttp.OkHttpClientImpl$OkHttpAsyncBody.doConsume(OkHttpClientImpl.java:133)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	... 1 more
```

--- a/src/main/kotlin/com/redhat/devtools/intellij/kubernetes/model/AllContexts.kt
+++ b/src/main/kotlin/com/redhat/devtools/intellij/kubernetes/model/AllContexts.kt
@@ -126,7 +126,6 @@ open class AllContexts(
 	override fun setCurrentNamespace(namespace: String): IActiveContext<out HasMetadata, out KubernetesClient>? {
 		val old = this.current ?: return null
 		val newClient = clientFactory.invoke(namespace, old.context.name)
-		throwIfNotAccessible(namespace, newClient.get())
 		val new = setCurrentContext(newClient, old.getWatched())
 		if (new != null) {
 			modelChange.fireCurrentNamespaceChanged(new, old)

--- a/src/main/kotlin/com/redhat/devtools/intellij/kubernetes/model/context/ActiveContext.kt
+++ b/src/main/kotlin/com/redhat/devtools/intellij/kubernetes/model/context/ActiveContext.kt
@@ -119,9 +119,16 @@ abstract class ActiveContext<N : HasMetadata, C : KubernetesClient>(
         return if (!current.isNullOrEmpty()) {
             current
         } else {
-            val allNamespaces = getAllResources(namespaceKind, NO_NAMESPACE)
-            val namespace = allNamespaces.find { namespace:HasMetadata -> DEFAULT_NAMESPACE == namespace.metadata.name } ?: allNamespaces.firstOrNull()
-            return namespace?.metadata?.name
+            return try {
+                val allNamespaces = getAllResources(namespaceKind, NO_NAMESPACE)
+                val namespace =
+                    allNamespaces.find { namespace: HasMetadata -> DEFAULT_NAMESPACE == namespace.metadata.name }
+                        ?: allNamespaces.firstOrNull()
+                namespace?.metadata?.name
+            } catch (e: ResourceException) {
+                logger<ActiveContext<*,*>>().warn("Could not list all namespaces to use 1st as current namespace.", e)
+                null
+            }
         }
     }
 

--- a/src/test/kotlin/com/redhat/devtools/intellij/kubernetes/model/AllContextsTest.kt
+++ b/src/test/kotlin/com/redhat/devtools/intellij/kubernetes/model/AllContextsTest.kt
@@ -384,20 +384,6 @@ class AllContextsTest {
 		assertThat(contextName.allValues[1]).isEqualTo(activeContext.context.name)
 	}
 
-	@Test(expected = ResourceException::class)
-	fun `#setCurrentNamespace(namespace) should throw if current namespace is forbidden`() {
-		// given
-		val client = client(KubernetesClientException("a disturbance in the force")) // throws upon client#namespaces
-		val clientConfig = clientConfig(currentContext, contexts, configuration)
-		val clientAdapter = clientAdapter(clientConfig, client) // no config so there are no contexts
-		val clientFactory = clientFactory(clientAdapter)
-		val allContexts = TestableAllContexts(modelChange, contextFactory, clientFactory)
-		// when
-		allContexts.setCurrentNamespace("dark side")
-		// then
-		verify(modelChange, never()).fireCurrentNamespaceChanged(anyOrNull(), anyOrNull())
-	}
-
 	@Test
 	fun `#onKubeConfigChanged() should NOT fire if new config is null`() {
 		// given

